src/screens/OrganizationVenues/OrganizationVenues.spec.tsx

```
@@ -215,14 +215,96 @@ describe('Organisation Venues', () => {
+                    tabIndex={-1}
+                    type="button"
+                  >
+                    Search
+                  </button>
+                </>
+              )}
+            </div>
+
+            {sorting?.map((sort, index) => (
+              <div key={index}>
+                <button title={sort.title} data-testid={sort.testIdPrefix}>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)